internal/rpc/client/library/client.go

```diff
@@ -0,0 +1,37 @@
+package library
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/aquasecurity/trivy/internal/rpc/client"
+
+	"github.com/aquasecurity/trivy/internal/rpc"
+	"github.com/aquasecurity/trivy/pkg/types"
+
+	ptypes "github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/trivy/rpc/detector"
+	"golang.org/x/xerrors"
+)
+
+type DetectClient struct {
+	token  string
+	client detector.LibDetector
+}
+
+func NewDetectClient(remoteURL, token string) DetectClient {
+	client := detector.NewLibDetectorProtobufClient(remoteURL, &http.Client{})
```

Comment:
Rename this to something other than `client`, currently it collides with the package name.

Reply:
I changed the name here.
https://github.com/aquasecurity/trivy/pull/303/commits/fe3dbfcb992706873175b40f001fcac962b30a7e#diff-ab99057db778dd939dd5eb08395787b8R38